distribution/src/resources/drill-config.sh

```diff
@@ -366,11 +366,18 @@ if [ $? -eq 0 ]; then
   fatal_error "Java 1.7 or later is required to run Apache Drill."
 fi
 
+# Create Drill local udf area, if it does not exist
+DRILL_UDF_DIR="$DRILL_HOME/udf"
+if [[ ! -d "$DRILL_UDF_DIR" ]]; then
+  mkdir -p "$DRILL_UDF_DIR"
+fi
+
```

Comment:
This should be $DRILL_CONF_DIR/udf, not $DRILL_HOME. $DRILL_CONF_DIR points to the user's "site" directory, which will be outside of $DRILL_HOME for Drill-on-YARN.

Not sure we need to create this directory. The pattern followed elsewhere is to include the directory on the class path if it exists.

In Drill-on-YARN, we CAN'T create the directory; the "localized" Drill and site directories are read-only.